lynx/plan.sh

```diff
@@ -0,0 +1,23 @@
+pkg_origin=core
+pkg_name=lynx
+pkg_version=2.8.8
+pkg_description="Lynx is the text web browser."
+pkg_upstream_url=http://lynx.browser.org/
+pkg_license=("GPL-2.0")
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_source="http://invisible-mirror.net/archives/$pkg_name/tarballs/$pkg_name${pkg_version}rel.2.tar.gz"
```

Comment:
I tend to leave it off unless it's needed (in this case the last one would have read `$pkg_versionrel` which is not what I want.) ShellCheck doesn't seem to have a preference.